float32 angle_in_low
float32 angle_in_zero
float32 angle_in_high

float32 angle_out_low
float32 angle_out_zero
float32 angle_out_high

float32 speed_in_low
float32 speed_in_zero
float32 speed_in_high

float32 speed_out_low
float32 speed_out_zero
float32 speed_out_high


